### PR TITLE
Add dfviz

### PR DESCRIPTION
**Pull request by martindurant**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/dfviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - panel >=0.5.1
+    - hvplot
+
+test:
+  imports:
+    - dfviz
+
+about:
+  home: https://github.com/intake/dfrviz
+  license: BSD-3-Clause
```
Comment (xhochy) [inline recipes/dfviz/meta.yaml]: The attached LICENSE is Apache but here you state BSD.

File: recipes/dfviz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }}  -m pip install . --no-deps --ignore-installed -vv"
```
Comment (xhochy) [inline recipes/dfviz/meta.yaml]: The arguments are no longer required by supplied by default.
```suggestion
  script: "{{ PYTHON }}  -m pip install . -vv"
```

File: recipes/dfviz/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+{% set sha256 = "f8fd80ea7114ba71e575104b5648e0e52e0a677f8c11162d2ee1100c51898d42" %}
+
+package:
+   name:  {{ name }}
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/dfviz/meta.yaml]: Place the sha256 directly here

**Comment by martindurant** [comment]
Thanks @xhochy

**Pull request merged**